### ISSUE TITLE
Please support with-contenv shebang from s6-overlay

### ISSUE THREAD

**Issue by nemchik** [koalaman/shellcheck#1760]
#### For new checks and feature suggestions
- [ ] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/usr/bin/with-contenv sh
echo "$MYENV"
```

#### Here's what shellcheck currently says:

From shellcheck.net
```
$ shellcheck myscript
 
Line 1:
#!/usr/bin/with-contenv sh
^-- SC1008: This shebang was unrecognized. ShellCheck only supports sh/bash/dash/ksh. Add a 'shell' directive to specify.

$
```

#### Here's what I wanted or expected to see:

It would be nice if this did not require ignoring SC1008.

Ref links
https://github.com/just-containers/s6-overlay/blob/master/README.md#container-environment
https://github.com/just-containers/s6-overlay/blob/master/builder/overlay-rootfs/usr/bin/with-contenv

**Comment by nemchik** [comment]
Just an FYI, s6 overlay has changed from
`/usr/bin/with-contenv` to `/command/with-contenv` although they offer a symlink for the original path, so it would be great if both could be supported.
The new file location in their repository is https://github.com/just-containers/s6-overlay/blob/master/layout/rootfs-overlay/package/admin/s6-overlay-%40VERSION%40/command/with-contenv
